package.json

```diff
@@ -26,13 +26,16 @@
     "google storage",
     "storage"
   ],
+  "mocha": {
```

Comment:
What does moving this here vs the command line provide?

Reply:
It allows 2 things:
- `npx mocha <path-to-test(s)>` without additional flags to type/remember
- Easier IDE integration - IDEs no long require additional config in order to run `mocha` tests (which can change over time)

Reply:
Add, if `mocha` is globally installed (`npm i -g mocha`) simply typing `mocha` in the CLI will run the default tests (same as `npx mocha`)